ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -80,7 +80,7 @@ import qualified Shelley.Spec.Ledger.API as SL
 import qualified Shelley.Spec.Ledger.BaseTypes as SL (Seed (..),
                      mkNonceFromNumber, textToUrl)
 import qualified Shelley.Spec.Ledger.BlockChain as SL (TxSeq (..))
-import qualified Shelley.Spec.Ledger.Coin as SL (DeltaCoin (..))
+import qualified Cardano.Ledger.Coin as SL (DeltaCoin (..))
```

Comment:
Maybe worth changing the qualified name to something that does not resemble Shelley as this seems to have been moved away from `Shelley.Spec`. And possibly moving the import 8 lines upward to have it sorted.

Reply:
I've forgot to format it :) Crap.
Regarding rename.. Hmm .. I think `SL` stands for "Spec Ledger" but I might be wrong

Reply:
This got formatted